Samples/Sample XML/StudentEnrollment.xml

```diff
@@ -2367,7 +2521,7 @@
 		<EntryDate>2021-08-23</EntryDate>
 		<EntryGradeLevel>uri://ed-fi.org/GradeLevelDescriptor#First grade</EntryGradeLevel>
 		<GraduationPlanReference ref="GPLN_Minimum-GBES"/>
-		<FullTimeEquivalency>0.75</FullTimeEquivalency>
+		<FullTimeEquivalency>1.00</FullTimeEquivalency>
```

Comment:
These existing 0.75 values probably shouldn't be defaulted to 1.00.

Reply:
restored original value.